ci-operator/jobs/rhpit/interop-tests/rhpit-interop-tests-main-periodics.yaml

```
@@ -1,4 +1,79 @@
 periodics:
+- agent: kubernetes
+  cluster: build07
+  cron: 0 6 25 10 *
```

Same comment on cron value (generated from `make update`):
```suggestion
  cron: 0 23 31 2 *
```